pkg/proto/snapshot_types.go

```diff
@@ -121,6 +150,28 @@ type DataEntriesSnapshot struct { // AccountData in pb
 
 func (s DataEntriesSnapshot) Apply(a SnapshotApplier) error { return a.ApplyDataEntries(s) }
 
+func (s DataEntriesSnapshot) Equal(otherSnapshot AtomicSnapshot) (bool, error) {
```

Comment:
It's necessary to say that this function mutates passed argument, i.e. elements order in the`DataEntries` slice.